Library/Formula/jhead.rb

```diff
@@ -1,14 +1,16 @@
-require 'formula'
-
 class Jhead < Formula
-  homepage 'http://www.sentex.net/~mwandel/jhead/'
-  url 'http://www.sentex.net/~mwandel/jhead/jhead-2.97.tar.gz'
-  sha1 'ca4965a19d60078a3fe2cfb6d3635a083f958f2e'
+  homepage "http://www.sentex.net/~mwandel/jhead/"
+  url "http://www.sentex.net/~mwandel/jhead/jhead-3.00.tar.gz"
+  sha1 "6bd3faa38cc884b5370e8e8f15bc10cbb706ec7a"
 
   def install
     system "make"
     bin.install "jhead"
```

Comment:
I can't provide a link to the upstream patch submission. All communication with the upstream maintainer goes via email. I'm currently in an email exchange with the maintainer to see if he's willing to accept the patch.

As for the patch itself: it is possible to use the patch in the formula, but given the size of the patch, the current `install` block, and the frequency with which jhead gets an update, I think it's a bit of an overkill.

Let's wait a bit until I get a definitive answer from the maintainer.